atlascope/core/management/populate/jobs.json

```diff
@@ -24,9 +24,6 @@
         "job_type": "nucleus_to_gland_distance",
         "id": 4003,
         "investigation": "Analytics playground",
-        "original_dataset": "HTA9_1_BA_F_ROI04",
-        "additional_inputs": {
-            "gland_map_path": "atlascope/core/management/populate/inputs/gland_map.jpg"
-        }
+        "original_dataset": "HTA9_1_BA_F_ROI04"
```

Comment:
Why dump the gland map input? (I suppose this is connected to why we're introducing random gland map generation in this PR.)